bestie/proto/test_metrics.proto

```diff
@@ -5,6 +5,42 @@ package bestie.proto;
 
 option go_package = "github.com/enfabrica/enkit/bestie/proto";
 
+// Each metric tag must be supplied in the form of a key/value string pair.
+message KeyValuePair {
+    string key = 1;
+    string value = 2;
+}
+
+// Specialized BigQuery test result definition compatible with Prometheus remote read adapter.
+//
+// This message is similar to TestMetric, but with added restrictions to make it
+// less ambiguous for the message sender to construct properly. The same BigQuery
+// table holds both TestMetric and ResultMetric rows from a variety of test applications
+// (clients), therefore standardizing on a specific metricname ('testresult') is
+// essential. This name, a corresponding 'result' tag, and a numeric value is inserted
+// by the BES Endpoint service as it processes a ResultMetric message, therefore these are
+// not specifically defined by the message fields.
+//
+// The allowed Result values are PASS or FAIL, with INCOMPLETE indicating a test
+// that was intended to be run, but could not due to some extenuating condition, for
+// example reliance upon a specific DUT or traffic generator that was unavailable.
+message ResultMetric {
+    enum Type {
+        SUMMARY = 0; // default, if field not specified
+    }
+    Type type = 1;
+    enum Result {
+        FAIL = 0;
```

Comment:
Typically with enums you want an UNKNOWN as the 0 value so that if the client forgets to set it it's very obvious